Sources/Logging/Protocols/Logger.swift

```diff
@@ -8,42 +8,36 @@
 
 protocol Logger {
 
-    // MARK:- Destination Management
-
-    func register(_ destination: LogDestination)
-    func unregister(_ destination: LogDestination)
-    func removeAllDestinations()
-
     // MARK:- Logging
 
     func verbose(_ message: @autoclosure () -> String,
                  file: StaticString,
-                 function: String,
+                 function: StaticString,
                  line: UInt)
 
     func debug(_ message: @autoclosure () -> String,
                file: StaticString,
-               function: String,
+               function: StaticString,
                line: UInt)
 
     func info(_ message: @autoclosure () -> String,
               file: StaticString,
-              function: String,
+              function: StaticString,
               line: UInt)
 
     func warning(_ message: @autoclosure () -> String,
                  file: StaticString,
-                 function: String,
+                 function: StaticString,
                  line: UInt)
 
     func error(_ message: @autoclosure () -> String,
                file: StaticString,
-               function: String,
+               function: StaticString,
                line: UInt)
     
     func log(level: Log.Level,
```

Comment:
Is this necessary to be public? i think this should be private, no? we have helper methods for all the levels, so no need for this one.